environment/src/test/java/com/sequenceiq/environment/environment/validation/validators/PublicKeyValidatorTest.java

```diff
@@ -0,0 +1,70 @@
+package com.sequenceiq.environment.environment.validation.validators;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import org.junit.jupiter.api.Test;
+
+import com.sequenceiq.cloudbreak.validation.ValidationResult;
+
+class PublicKeyValidatorTest {
```

Comment:
based on the regex we should cover more scenario

Reply:
I'll create one for each algorithm

Reply:
I see these "ssh-rsa", "ecdsa-sha2-nistp256", "ecdsa-sha2-nistp384", "ecdsa-sha2-nistp521", "ssh-ed25519", "ssh-dss"

Reply:
those are the algorithms